docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx

```diff
@@ -663,6 +667,11 @@ class ComponentExample extends React.Component<ComponentExampleProps, ComponentE
         <Divider section horizontal />
       </Visibility>
     )
+
+    if (excludedThemes && excludedThemes.indexOf(themeName) > -1) {
```

Comment:
In general, this check should be done at the beginning of the function - all the information are available there, there is no need to compute the whole result and then return ''.